src/ContactsTable.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import Contact from './Contact';
+
+class ContactsTable extends Component {
+  constructor(props) {
```

Comment:
I think you need to rearrange the state so that you can add new one. You don't have access to all the data from this component